reference/api/documents.mdx

```diff
@@ -248,6 +248,54 @@ An array of documents. Each document is represented as a JSON object.
 ]
 ```
 
+#### `_vectors`
+
+`_vectors` is a special document attribute containing vector embeddings for AI-powered search. It must be an object.
+
+Each key of the `_vectors` object must be the name of a configured embedder and correspond to an array of numbers:
+
+```json
+[
+  {
+    "id": 452,
+    "title": "Female Trouble",
+    "overview": "Delinquent high school student Dawn Davenport runs away from home and embarks upon a life of crime.",
+    "_vectors": {
+      "default": [0.003, 0.1, 0.75],
+      "ollama": [0.1, 0.2, 0.3]
+    }
+  }
+]
+```
+
+Keys of the `_vectors` object may also correspond to an object with two fields, `embeddings` and `regenerate`:
+
+```json
+[
+  {
+    "id": 452,
+    "title": "Female Trouble",
+    "overview": "Delinquent high school student Dawn Davenport runs away from home and embarks upon a life of crime.",
+    "_vectors": {
+      "default": {
+        "embeddings": [0.1, 0.2, 0.3],
+        "regenerate": true
+      }
+    }
+  }
+]
+```
+
+`embeddings` must be an array of numbers representing a single embedding for that document. It may also be an array of arrays of numbers representing multiple embeddings for that document. `embeddings` is an optional field.
+
+`regenerate` must be a Boolean value. If `regenerate` is `true`, Meilisearch automatically generates embeddings for that document immediately and every time the document is updated. If `regenerate` is `false`, Meilisearch keeps the last value of the embeddings on document updates.
+
+If `_vectors` is empty or `null`, Meilisearch treats the document as not having any embeddings. This document is then ignored during AI-powered searches.
```

Comment:
> If `_vectors` is empty or `null`, Meilisearch treats the document as not having any embeddings.

I don't think that's true. I think this paragraph can probably be removed since we have the contents of the capsule already